Show party hydra credentials as hex in TUI

ch1bo opened the pull request:
Before:
![image](https://user-images.githubusercontent.com/2621189/167078888-833347a4-3d28-40e8-b17c-f8f480b06c6c.png)

After:
![image](https://user-images.githubusercontent.com/2621189/167078907-44fd42e4-13eb-4433-a637-75be3bf1e79d.png)

ch1bo committed:
Add a known issue about a transient error displayed on fanout

ch1bo committed:
Add SerialiseAsRawBytes instance to Hydra.VerificationKey and use it in TUI

This allows to conveniently show the hex representation of Parties and
their Hydra credentials

KtorZ reviewed (approved):
Nice. We could even truncate it and only show the last 8 digits or so, for most (all?) demo, that would do!

ch1bo committed:
Add assertion to hydra-tui spec about ownParty rendering

ch1bo committed:
Update TUISpec to assert on unescaped bytes

This makes usage of shouldRender easier as it does not require us to add
escape codes to the expected bytes.
